tools/cldr-code/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -955,6 +958,25 @@ public void writePackagingFiles(String outputDir, String packageName) throws IOE
         writeReadme(outputDir, packageName);
     }
 
+    /**
+     * Write the ## License section
+     */
+    public void writeCopyrightSection(PrintWriter out) {
+        out.println(CldrUtility.getCopyrightMarkdown() + "\n" +
+        "A copy of the license is included as [LICENSE](./LICENSE).");
+    }
+
+    /**
+     * Write the readme fragment from cldr-json-readme.md plus the copyright
+     * @param outf
+     * @throws IOException
+     */
+    private void writeReadmeSection(PrintWriter outf) throws IOException {
+        FileCopier.copy(CldrUtility.getUTF8Data("cldr-json-readme.md"), outf);
+        outf.println();
+        writeCopyrightSection(outf);
```

Comment:
writeCopyrightSection could be private, and could be moved below its caller, writeReadmeSection

likewise writeReadmeSection could be moved below its callers

Reply:
Ok .  Would like to cleanup in a future PR if this is not critical